### PR TITLE
Move browser tab rebase

### PR DESCRIPTION
**Pull request by zuphilip**

### CODE REVIEW

File: src/chrome/content/scaffold/scaffold.xul [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+	
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Affero General Public License for more details.
+	
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero.  If not, see <http://www.gnu.org/licenses/>.
+	
+	***** END LICENSE BLOCK *****
```
Comment (adomasven) [inline src/chrome/content/scaffold/scaffold.xul]: This looks unnecessary

Comment (adomasven) [reply src/chrome/content/scaffold/scaffold.xul]: Oh, I guess since you've replaced spaces with tabs for the whole file that's fine.

**Comment by adomasven** [comment]
So with pull requests you can just rebase the existing branch and then force push. I.e.
```bash
git checkout master
git pull master
git checkout move-browser-tab
git rebase master
# Solve conflicts
git push -f
```

File: src/chrome/content/scaffold/scaffold.xul (hunk 1 above)
Comment (dstillman) [reply src/chrome/content/scaffold/scaffold.xul]: The rest of the file is just indented an additional tab stop. That can be done, but it should be in a separate commit, and this part should be reverted.

Comment (dstillman) [reply src/chrome/content/scaffold/scaffold.xul]: Oh well, too late.

Comment (adomasven) [reply src/chrome/content/scaffold/scaffold.xul]: This PR is two commits. One fixes the spacing.

Comment (dstillman) [reply src/chrome/content/scaffold/scaffold.xul]: Ah, missed that. (The license block should still be left alone for other files, though.)

Comment (zuphilip) [reply src/chrome/content/scaffold/scaffold.xul]: Okay. Next time, I will try to do better. 😇

**Comment by zuphilip** [comment]
@adomasven Yes, these are the steps I wanted to do. But solving the conflicts were difficult here (because the changes of spaces showed more or less everything as a block from master and this branch). Thus, after I will I guessed that is easier to redo the commits on top of master.

**Comment by zuphilip** [comment]
I forgot to do this (but we can do that later):

>> Now, we might want to delete one of the Current URL. I suggests to have that in the browser pane only.
>
> Agreed.

**Pull request merged**